GameVersion.h

```diff
@@ -43,7 +43,7 @@ class GameVersion: commonItems::parser
 	int fourthPart = 0;
 };
 
-class GameVersion::Factory: parser
+class GameVersion::Factory: commonItems::parser
```

Comment:
GameVersion is in the commonItems namespace, so it's unnecessary to specify here.